bigtable/integration_test.go

```diff
@@ -4363,31 +4379,73 @@ func setupIntegration(ctx context.Context, t *testing.T) (_ IntegrationEnv, _ *C
 	}, nil
 }
 
+func deleteTableWithRetry(ctx context.Context, t *testing.T, adminClient *AdminClient, tableName string) error {
+	return retry(t, func() error { return adminClient.DeleteTable(ctx, tableName) },
+		nil)
```

Comment:
What happens when `nil` is passed in as a function?

Reply:
Handled nil